metrics_intro.md

```diff
@@ -10,6 +10,12 @@ Once metrics are ingested into Operations for Applications, administrators can [
 
 All users can then receive [alerts](https://docs.wavefront.com/alerts.html), drill down into the prepared dashboards and charts, and start finding the root cause for problems.
 
+In terms of retention period, there are two metric types:
```

Comment:
The retention period varies based on the metric type:
  * Persistent metrics are retained for 18 months. By default, all ingested metrics are persistent but convertible.
  * Ephemeral metrics are retained for 28 days. By default, only the [internal metrics](https://docs.wavefront.com/wavefront-internal-metrics.html) are ephemeral and they are not convertible.